lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -346,7 +348,6 @@ class ImageElements extends Gatherer {
       // or it's not in the top 50 largest images.
       if (
         (element.isPicture || element.isCss || element.srcset) &&
-        networkRecord &&
```

Comment:
confusion fix, we always default `networkRecord` to `{}` if we couldn't find one so this check does nothing